gimie/parsers/__init__.py

```diff
@@ -72,6 +74,8 @@ def select_parser(
     # Only parse licenses in the root directory
```

Comment:
suggestion: #only parse licenses & CFF files in the root directory

Reply:
addressed in https://github.com/sdsc-ordes/gimie/pull/107/commits/652a59940a26fa7d946d620d22023d01e88bcd7a